### PR TITLE
Bump psammead dependencies

### LINKED ISSUE #2484: Large amount of bundle is duplicated psammead packages
**Describe the bug**
About 60kb of the simorgh bundle is duplicated psammead packages. This is due to psammead packages not be kept up to date.

**To Reproduce**
Checkout https://github.com/bbc/simorgh/pull/2321 and `npm run build` to see detail.

**Expected behavior**
Build passes on that PR or just stop duplicating packages.

### PR DESCRIPTION
**Pull request by thekp**
Resolves: #2484

**Overall change:** _Bumps Psammead dependencies_

**Code changes:**

- _update package versions_

**REGRESSION REQUIRED**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Comment by jamesbhobbs** [comment]
Re the snyk vuln, it's not great but as it's in webpack only there's no attack vector for https://app.snyk.io/org/news-articles/test/github/35916eba-8cf2-4204-89f8-45455a5b9d95/c0ba09b42ad579d131b1d2c087939c94619077f5..c897dac0acdf0394797828d9bf147c0930143299?fromStatus=true&fromGitHubAuth=true so far as I can tell.

File: package.json [hunk 1]
```diff
@@ -124,6 +124,7 @@
     "express-static-gzip": "^1.1.3",
     "gnu-terry-pratchett": "0.0.2",
     "helmet": "^3.14.0",
+    "inspectpack": "^4.2.1",
```
Comment (jamesbhobbs) [inline package.json:126]: Shouldn't be a prod dep, but is due to https://github.com/bbc/simorgh/issues/2531 so it's ok here.

**Review by jamesbhobbs — APPROVED** [review]

File: scripts/bundleSize.sh [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
-min=605
-max=625
+min=500
```
Comment (dr3) [inline scripts/bundleSize.sh]: Do we really have a +/- 5K of 100K?!

Comment (thekp) [reply scripts/bundleSize.sh]: nope, I just put that as a placeholder for now. Happy to change to any suggestion.

**Review by dr3 — CHANGES_REQUESTED** [review]
Navigation needs to be passed `service`
Timestamp-container needs to be passed `service` in storypromo and articletimestamp containers

File: scripts/bundleSize.sh (hunk 2 above)
Comment (dr3) [reply scripts/bundleSize.sh]: Min needs to be 5K less than the smallest one, and Max 5K more than the biggest

**Review by dr3 — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to pod testers.

**Pull request merged**